### PR TITLE
React SPA

### PR DESCRIPTION
**Pull request by vishalrishu**
- Basic setup
- Routes added
- Data fetching and rendering added

### CODE REVIEW

File: app/javascript/components/EditInterview.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import React, {useState, useEffect} from "react";
+import { Link } from "react-router-dom";
+
+const EditInterview = (props)=> {
+
+  const [start_time, setStarttime] = useState('');
+  const [end_time, setEndtime] = useState('');
+  const [description, setDescription] = useState('');
+  const [participant_ids, setParticipants] = useState([]);
```
Comment (joydeep-ib) [inline app/javascript/components/EditInterview.js:9]: Better use camelCase

Comment (vishalrishu) [reply app/javascript/components/EditInterview.js:9]: ok

Comment (aswanth9495) [inline app/javascript/components/EditInterview.js:9]: You can have one state for storing all of this right

File: app/javascript/components/ShowInterview.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      .then(res => res.json())
+      .then(data => {setInterview(data)})
+  }, [])
+
+  return (
+    <div>
+      {!interview ? (
+        ""
+      ) : (
+        <div>
```
Comment (aswanth9495) [inline app/javascript/components/ShowInterview.js:19]: You can do this as well 
```
!interview && (rest of the component)

File: app/javascript/components/EditInterview.js (hunk 1 above)
Comment (vishalrishu) [reply app/javascript/components/EditInterview.js:9]: Yes.

**Pull request merged**